gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-default-page/api-documentation-v4-default-page.component.ts

```diff
@@ -83,7 +83,7 @@ export class ApiDocumentationV4DefaultPageComponent implements OnInit, OnDestroy
   }
 
   editPage(pageId: string) {
-    this.router.navigate([pageId], {
+    this.router.navigate(['homepage/' + pageId], {
```

Comment:
This is fine, but just `'homepage'` works. Angular will add the `"/"` between the items

Reply:
No, if I do it I get an error:
 NG04002: Cannot match any routes. URL Segment: 'default/apis/0643f7cd-bad0-4f09-83f7-cdbad06f0955/v4/documentation/default-pages/homepage5066cf3f-7817-45e7-a6cf-3f7817b5e747'

Reply:
@jourdiw can we please merge it?

Reply:
ah I see. It should be:

```suggestion
    this.router.navigate(['homepage', pageId], {
```

Reply:
Done